neo/Wallets/WalletIndexer.cs

```diff
@@ -218,31 +219,29 @@ private void ProcessBlock(Block block, HashSet<UInt160> accounts, WriteBatch bat
                 {
                     foreach (UInt160 account in accounts_changed)
                         batch.Put(SliceBuilder.Begin(DataEntryPrefix.ST_Transaction).Add(account).Add(tx.Hash), false);
-                    WalletTransaction?.Invoke(null, new WalletTransactionEventArgs
-                    {
-                        Transaction = tx,
-                        RelatedAccounts = accounts_changed.ToArray(),
-                        Height = block.Index,
-                        Time = block.Timestamp
-                    });
+                    change_set.Add((tx, accounts_changed.ToArray()));
                 }
             }
+            return change_set.ToArray();
```

Comment:
Is there any advantage here of converting to an array instead of just returning the list and using the list? Seems like an extra unnecessary conversion.